src/main/java/com/atlassian/jira/cloud/jenkins/config/JiraCloudPluginConfig.java

```diff
@@ -21,17 +21,35 @@
 @Extension
 public class JiraCloudPluginConfig extends GlobalConfiguration {
 
+    public static final String FIELD_NAME_AUTO_BUILDS = "autoBuilds";
```

Comment:
The changes in this file are to extend the Jenkins configuration UI and provide the new "send builds automatically" and "send deployments automatically" checkboxes and input fields for the regexes.